### ISSUE TITLE
RFE: Add documentation for charts

### ISSUE THREAD

**Issue by jsfan3** [codenameone/CodenameOne#3029]
The Javadoc of the various charts (BarChart, BubbleChart, CombinedXYChart, CubicLineChart, DialChart, DoughnutChart, LineChart, PieChart, RadarChart, RangeBarChart, RoundChart, ScatterChart, TimeChart) is empty, there is no documentation of usage...

Can you add documentation and examples of usage?

Thank you

**Comment by Manish254252** [comment]
Hi ! Anyone working on this ?

**Comment by shai-almog** [comment]
Not that I know @Manish254252

**Comment by Manish254252** [comment]
If no one is working this please assign this to me . @shai-almog

**Comment by shai-almog** [comment]
Done @Manish254252

**Comment by aishuarya** [comment]
Hi @shai-almog 

Is someone working on this ?

**Comment by shai-almog** [comment]
@aishuarya no. Feel free to take it.

**Comment by aishuarya** [comment]
@shai-almog 

Could you provision enough permissions to work on this repo?

**Comment by shai-almog** [comment]
@aishuarya no. You fork the project and work in your own repository. Make the changes there and then submit a pull request from your fork to the main repository.

**Comment by shai-almog** [comment]
https://github.com/codenameone/CodenameOne/pull/4047
